React InstantSearch/routing-seo-friendly/src/App.js

```diff
@@ -1,11 +1,14 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
```

Comment:
We do use the hooks inside storybook but we've never used them for documentation purpose. I'm wondering if it's the right time to do the changes. It's a different paradigm that might be complicated to understand at first from our users. I would have wait a bit more for their introduction into the doc.